server/server.go

```diff
@@ -768,7 +828,7 @@ func (s *Server) AcceptLoop(clr chan struct{}) {
 		s.Noticef("TLS required for client connections")
 	}
 
-	s.Debugf("Server id is %s", s.info.ID)
+	s.Noticef("Server id is %s", s.info.ID)
```

Comment:
We should do clustername too.. And make sure all servers in a cluster are consistent?

Reply:
This is the client accept loop. I could put it here or in the gateway accept loop.